neo/Network/RPC/RpcServer.cs

```diff
@@ -203,12 +203,6 @@ private JObject Process(string method, JArray _params)
                     {
                         return GetVersion();
                     }
-                case "invoke":
```

Comment:
This will be added in the future, i guess?

Reply:
No. In NEO 3.0, we need to unify the entry points. So users should use `invokefunction`.

https://github.com/neo-project/neo/blob/a9fb6f808bbf32a8c4e51d7a25c42b903fd81d29/neo/Network/RPC/RpcServer.cs#L206-L212